x-pack/plugins/observability/server/routes/slo/route.ts

```diff
@@ -303,6 +305,29 @@ const getSloDiagnosisRoute = createObservabilityServerRoute({
   },
 });
 
+const getSloBurnRates = createObservabilityServerRoute({
+  endpoint: 'POST /internal/observability/slos/{id}/_burn_rates 2023-06-14',
+  options: {
+    tags: [],
```

Comment:
We want to restrict the access to slo read at least: `tags: ['access:slo_read']`